variables.tf

```diff
@@ -14,10 +14,15 @@ variable "location" {
   }
 }
 
+variable "existing_resource_group" {
```

Comment:
we got some feedback to ask us to rename this to `use_existing_resource_group` - you might want to do the same for consistency?